benchmark/strong_scaling_shallow_water_model_single.jl

```diff
@@ -0,0 +1,44 @@
+using Logging
+using MPI
+using JLD2
+using BenchmarkTools
+
+using Oceananigans
+using Oceananigans.Distributed
+using Benchmarks
+
+Logging.global_logger(OceananigansLogger())
+
+MPI.Init()
+comm = MPI.COMM_WORLD 
+
+local_rank = MPI.Comm_rank(comm)
+R = MPI.Comm_size(comm)
+
+Nx = parse(Int, ARGS[1])
+Ny = parse(Int, ARGS[2])
+
+@info "Setting up distributed incompressible model with N=($Nx, $Ny) grid points on $R rank(s)..."
+
+topo = (Periodic, Periodic, Bounded)
+distributed_grid = RegularRectilinearGrid(topology=topo, size=(Nx, Ny, 1), extent=(1, 1, 1))
+arch = MultiCPU(grid=distributed_grid, ranks=(1, R, 1))
+model = DistributedShallowWaterModel(architecture=arch, grid=distributed_grid)
+
+@info "Warming up distributed shallow water model..."
+
+time_step!(model, 1) # warmup
+
+@info "Benchmarking distributed shallow water model..."
+
+trial = @benchmark begin
+    @sync_gpu time_step!($model, 1)
+    MPI.Barrier(comm)
+end samples=10
+
+@info "Rank $local_rank is done benchmarking!"
+
+jldopen("strong_scaling_shallow_water_model_$(R)_$local_rank.jld2", "w") do file
```

Comment:
Not sure if it caused the failure but this should probably be changed (I accidently merged it into `main`...).

```suggestion
jldopen("strong_scaling_shallow_water_model_$R.jld2", "w") do file
```

Reply:
Actually, I think you do need this since the data files end with say `...model_4_3.jld2` and so on.

Reply:
I have done this in `strong_scaling_shallow_water_model.jl` using a double loop.  It seems to work but I recognize there might be a better way of doing this

Reply:
I used the following and it seemed to work
```
for r in ranks, local_rank in collect(0:(r-1))
    jldopen(file_name, "r") do file suite[((Nx, Ny), r)] = file["trial"]
    end
end
```